installer/data/mysql/atomicupdate/bug_19575-use-canonical-field-names-and-resolve-aliased-fields.sql

```diff
@@ -7,7 +7,7 @@ UPDATE `search_field` SET `name` = 'control-number', `label` = 'control-number'
 UPDATE `search_field` SET `name` = 'place-of-publication', `label` = 'place-of-publication' WHERE `name` = 'place';
 UPDATE `search_field` SET `name` = 'date-of-acquisition', `label` = 'date-of-acquisition' WHERE `name` = 'acqdate';
 UPDATE `search_field` SET `name` = 'isbn', `label` = 'isbn' WHERE `name` = 'isbn';
-UPDATE `search_field` SET `name` = 'authority-number', `label` = 'authority-number' WHERE `name` = 'an';
+UPDATE `search_field` SET `name` = 'koha-auth-number', `label` = 'koha-auth-number' WHERE `name` = 'an';
```

Comment:
Is our atomicupdate able run these sql scripts? This is different from our the files we use.

Reply:
I got it to work and the stuff is in my dev install database but now I notice that this should also go to kohastructure.sql. Will submit a patch for that.